workbench-core/environments/src/utilities/environmentLifecycleHelper.ts

```diff
@@ -142,13 +142,13 @@ export default class EnvironmentLifecycleHelper {
 
   /**
    * Delete the access points for all datasets attached to a given workspace.
-   * @param envMetadata - the environment for which access point(s) were created
+   * @param environment - the environment for which access point(s) were created
```

Comment:
environment.DATASETS does not need to be defined (see Environment type). Does this handle that case?

Reply:
For consistency if you are going to rename this param, please rename the other params in this file that are used in the same way.

Reply:
Lodash returns `true` for `_.isEmpty(null)` and `_.isEmpty(undefined)` so we'll still get the desired behavior.